lesson_04_homework.rb

```diff
@@ -1,50 +1,106 @@
 # Code Reading
 
+# case is a type of conditional: 
 def fizzbuzz(num)
-  # Example comment line
 
+# use as many "when" statements as needed as long as match found
+# end with "else" statement if no match found
   case
+# when num is divided by 15 with no remainder, then it is true and you get "FizzBuzz" 
     when num % 15 == 0 then "FizzBuzz"
+# when num is divided by 3 with no remainder, then it is true and you get "Fizz" 
     when num % 3  == 0 then "Fizz"
+# when num is divided by 5 with no remainder, then it is true and you get "Buzz" 
     when num % 5  == 0 then "Buzz"
+# else it is false and you get num 
     else num
   end
 end
+puts fizzbuzz(30)
+puts fizzbuzz(9)
+puts fizzbuzz(50)
+puts fizzbuzz(4)
 
+
+# upto is a method
+# pass a value into (limit), 
+# print from 1 to the limit given
 def fizz_buzz_to(limit)
+# in increments of 1, "do" something each time you see "num"
   1.upto(limit).each do |num|
+# print each incremental number on each loop until (value passed to limit) is reached
     puts fizzbuzz(num)
+# end upto method
   end
+# end fizz_buzz_to
 end
 
+
 # Fix Broken Code
 
-def area_of_triangle(b, h)
-  puts b x height / 2
+# either change (b, h) to (base, height) throughout or to (b, h) throughout
+# def area_of_triangle(b, h)
+def area_of_triangle(base, height)
+# change x to *
+  # puts b x height / 2
+  puts base * height / 2
```

Comment:
@NancyGoodman, try not to added `puts` in the body of your methods - it makes them less flexible